db/migrations/02_add_username.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE peeps ADD COLUMN username VARCHAR(20);
```

Comment:
I think this is a good step, though I reckon it would be easier from a user standpoint, just to have the username be part of the `01_create_peeps_table.sql`